cmake/podioConfig.cmake.in

```diff
@@ -21,6 +21,7 @@
 set_and_check(podio_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
 set_and_check(podio_LIBRARY_DIR "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
 set_and_check(podio_PYTHON_DIR "@PACKAGE_podio_PYTHON_INSTALLDIR@")
+set(podio_IO_HANDLERS "@PODIO_IO_HANDLERS@")
 
 include(CMakeFindDependencyMacro)
 find_dependency(ROOT @ROOT_VERSION@)
```

Comment:
There needs to be something for  `find_dependency(SIO)` here, if enabled, no?

Reply:
I am actually not entirely sure if that is necessary. In the spirit of "it works on my machine" I have successfully built `edm4hep` with SIO support and also been able to read/write files without adding this.

Could it be that this is propagated through one of the targets that links to `SIO` by any chance? Like `podioSioIO` for example? Or is it enough that all the places where we would actually require to link to `SIO` are hidden inside cmake macros?

In the end it probably won't hurt to include it in any case, I suppose?

Reply:
Yes, at the moment the targets probably contain the full path to the SIO library and include directories used at build time. But if SIO moves to targets this would start failing.

Reply:
That makes sense. I have added this in the last commit.